src/state_title_screen.c

```diff
@@ -62,17 +64,17 @@ Gfx D_80077A50[] = {
 };
 
 typedef struct TitleDataStruct {
-    /* 0x0 */ s32 unk_00;
-    /* 0x4 */ s32 unk_04;
-    /* 0x8 */ s32 unk_08;
+    /* 0x0 */ s32 img_tile;
+    /* 0x4 */ s32 img_tile4b;
+    /* 0x8 */ s32 img_block;
     /* 0xC */ s32 img2_pal;
 } TitleDataStruct; // size = 0x10
 
 extern s16 D_800A0970;
-extern TitleDataStruct* D_800A0974;
-extern s32* D_800A0978;
-extern s32* D_800A097C;
-extern s32* D_800A0980;
+extern TitleDataStruct* TitleData;
```

Comment:
every symbol in this file should be prefixed with TitleScreen. so like TitleScreen_GfxData maybe